### DISCUSSION TITLE
How can I troubleshoot a .NET 8 RC1 install that seems to work but doesn't?

### DISCUSSION THREAD

**Discussion by tacosontitan** [dotnet/runtime#93173] [Q&A] [answered]
When I install RC1 for .NET 8 everything succeeds, and I can see it listed in the programs list when I go to "Add or remove programs", however, I can't use the SDK in my projects and running `dotnet --list-sdks` doesn't show that I have 8 at all.

What can I look into to troubleshoot this?

- I've already restarted my machine.
- I've uninstalled and reinstalled.
- I've verified it can be found as installed in "Add or remove programs".

**Comment by gfoidl** [comment, marked as answer]
Please see [It was not possible to find any installed .NET Core SDKs](https://learn.microsoft.com/en-us/dotnet/core/install/windows?tabs=net70#it-was-not-possible-to-find-any-installed-net-core-sdks).

Otherwise: Have you set any environment variables (beginning with `DOTNET_`)? If so they might point to somewhere where you .NET 8 installation isn't.

**Comment by tacosontitan** [comment]
Embarrassingly enough, I had installed the x86 version when I needed the x64 version.

**Reply by gfoidl** [reply]
Glad you found a solution 😃
